demos/face_detection/python/README.md

```diff
@@ -20,8 +20,8 @@ usage: face_detection.py [-h] [--input_images_dir INPUT_IMAGES_DIR]
                          [--server_cert SERVER_CERT]
                          [--client_cert CLIENT_CERT] [--client_key CLIENT_KEY]
 
-Demo for face detection requests via TFS gRPC API.analyses input images and
-saves with with detected faces.it relies on model face_detection...
+Demo for face detection requests via TFS gRPC API.Analyses input images and
+saves with with detected faces.It relies on model face_detection...
```

Comment:
```suggestion
Demo for face detection requests via TFS gRPC API analyses input images and
saves images with bounding boxes drawn around detected faces. It relies on face_detection model...
```

It looks like it has been copied from the script `--help`. Could you check out how it looks and make a change there as well if needed?